src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -239,6 +248,43 @@ internal ItemManager(RecentItemsManager recentItemsManager)
                 highlightedList,
                 completionHelper,
                 hasSuggestedItemOptions);
+
+
+            static bool ShouldBeFilteredOutOfCompletionList(VSCompletionItem item, ImmutableArray<CompletionFilter> activeNonExpanderFilters)
+            {
+                foreach (var itemFilter in item.Filters)
+                {
+                    if (activeNonExpanderFilters.Contains(itemFilter))
+                    {
+                        return false;
+                    }
+                }
+
+                return true;
```

Comment:
looks like a nice LINQ query. maybe it is worth to convert

Reply:
Fixed